packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { AlertBar } from '../AlertBar';
+import { ButtonIconReload } from '../ButtonIcon';
+import { useAlertBar } from 'fxa-settings/src/lib/hooks';
+import { useAccount, useLazyAccount } from '../../models';
+import { LinkExternal } from 'fxa-react/components/LinkExternal';
+
+const UTM_PARAMS =
+  '?utm_source=accounts.firefox.com&utm_medium=referral&utm_campaign=fxa-devices';
+const DEVICES_SUPPORT_URL =
+  'https://support.mozilla.org/kb/fxa-managing-devices' + UTM_PARAMS;
+
+export const Service = () => {
+  return (
+    <section
+      className="my-1"
+      id="service"
+      data-testid="settings-connected-service"
+    >
+      <div className="border-2 border-solid border-grey-100 rounded flex justify-around items-center">
+        <span>icon</span>
+
+        <div className="flex flex-col">
+          <h2>Service Name</h2>
+          <p className="text-sm">location</p>
+        </div>
+
+        <p>Active Now</p>
+
+        <button
+          className="cta-neutral cta-base disabled:cursor-wait whitespace-no-wrap"
+          data-testid="connected-service-sign-out"
+        >
+          Sign out
+        </button>
+      </div>
+    </section>
+  );
+};
+
+export const ConnectedServices = () => {
+  const alertBar = useAlertBar();
+  const [errorText, setErrorText] = useState<string>();
+  const onError = (e: Error) => {
+    setErrorText(e.message);
+    alertBar.show();
+  };
+  const [getAccount, { accountLoading }] = useLazyAccount((error) => {
+    setErrorText('Sorry, there was a problem refreshing the recovery key.');
+    alertBar.show();
+  });
+
+  return (
+    <section
+      className="mt-3"
+      id="connected-services"
+      data-testid="settings-connected-services"
+    >
+      <h2 className="font-header font-bold ml-4 mb-4">Connected Services</h2>
+      <div className="bg-white tablet:rounded-xl shadow py-4 px-6">
+        <div className="flex justify-between mb-4">
+          <p>Everything you are using and signed into.</p>
+          <ButtonIconReload
+            title="Refresh connected services"
+            classNames="hidden mobileLandscape:inline-block"
+            testId="connected-services-refresh"
+            disabled={accountLoading}
+            onClick={getAccount}
+          />
+        </div>
+
+        <Service />
+        <Service />
+        <Service />
+        <Service />
+        <Service />
+
+        <div className="mt-3 mb-4">
+          <LinkExternal
+            href={DEVICES_SUPPORT_URL}
+            className="link-blue text-sm mt-2 opacity-75 hover:opacity-100 focus:opacity-100"
```

Comment:
We should probably remove `opacity-75 hover:opacity-100 focus:opacity-100` so `link-blue` stays consistent with all of our links, unless the `link-blue` class should be modified 🤔 Also, `mt-2` doesn't appear to do anything - I think the parent instead of `mt-3` should be `mt-5`, the comps are showing `22px` space which we can round up to 24px.